src/common/types/antlr4/NaviLexer.js

```diff
@@ -3,79 +3,82 @@
 import antlr4 from 'antlr4';
 
 
-const serializedATN = [4,0,24,219,6,-1,2,0,7,0,2,1,7,1,2,2,7,2,2,3,7,3,2,
+const serializedATN = [4,0,26,227,6,-1,2,0,7,0,2,1,7,1,2,2,7,2,2,3,7,3,2,
```

Comment:
what even is this?

Reply:
To my knowledge, an ATN is a collection of graphs (think DFAs) that match text. These numbers are a serialized form of this.

https://www.antlr.org/api/JavaTool/org/antlr/v4/codegen/model/SerializedATN.html
https://stackoverflow.com/questions/41306505/antlr4-what-does-atn-stand-for

Reply:
Neat